artemis-dto/src/main/java/org/apache/activemq/artemis/dto/WebServerDTO.java

```diff
@@ -107,6 +107,9 @@ public class WebServerDTO extends ComponentDTO {
    @XmlAttribute
    public Integer scanPeriod;
 
+   @XmlAttribute
+   public Integer requestHeaderSize;
```

Comment:
My only comment here is that this attribute name, while matching the HttpConfiguration type's value isn't exactly self documenting as it doesn't really convey it is a limit unlike the other options in this DTO.  I'd have gone with something like requestHeaderSizeLimit or maxRequestHeaderSize just so it conveys the intent more clearly.

Reply:
I actually had the same thought.